extensions/cornerstone/src/OHIFCornerstoneViewport.js

```diff
@@ -102,6 +102,10 @@ class OHIFCornerstoneViewport extends Component {
   }
 
   getViewportData = async displaySet => {
+    if (!displaySet) {
```

Comment:
Add these lines in to prevent errors in the consoles when the displaySet had yet to be set.

Reply:
When would we pass the `OHIFCornerstoneViewport` a null/undefined displaySet? And why wouldn't this line return before this check?

https://github.com/OHIF/Viewers/blob/b748aaba1dcbe83cf04172489c2f8959dba3907d/extensions/cornerstone/src/OHIFCornerstoneViewport.js#L135

Reply:
See other comment.